Source/CesiumRuntime/Public/CesiumGeoreference.h

```diff
@@ -403,9 +450,9 @@ class CESIUMRUNTIME_API ACesiumGeoreference : public AActor {
 
   /**
    * @brief Computes the normal of the plane tangent to the surface of the
-   * ellipsoid that is used by this instance, at the provided position.
+   * ellipsoid that is used by this instance, at the provided position ECEF.
    *
-   * @param position The cartesian position for which to to determine the
+   * @param position The ECEF position for which to to determine the
    * surface normal.
    * @return The normal.
    */
```

Comment:
I made a whole bunch of changes like the one you suggest - and also removed this particular function entirely - in #1179. So I'm not going to make this change here because it will only create a conflict.

Reply:
Sounds good